Script/AtomicEditor/ui/frames/inspector/NodeInspector.ts

```diff
@@ -269,6 +269,8 @@ class NodeInspector extends ScriptWidget {
 
                     prefabComponent.breakPrefab();
 
+                    this.sendEvent("EditorActiveNodeChange", { node: this.node });
```

Comment:
We're not actually changing the active node here, and while this may reload the inspector UI now, it may not always be the case. (There is work to be done so that the Node/Component inspectors refresh)  

For now, this should disable, make invisible, or remove the prefab UI layout instead.